operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -630,10 +631,31 @@ public boolean wasInspectionRun() {
 
     private boolean isShouldContinue() {
       DomainPresenceInfo cachedInfo = getExistingDomainPresenceInfo(getNamespace(), getDomainUid());
+      int currentRetryCount = Optional.ofNullable(liveInfo)
+          .map(DomainPresenceInfo::getDomain)
+          .map(Domain::getStatus)
+          .map(DomainStatus::getRetryCount)
+          .orElse(0);
+
+      String existingError = Optional.ofNullable(liveInfo)
+          .map(DomainPresenceInfo::getDomain)
+          .map(Domain::getStatus)
+          .map(DomainStatus::getMessage)
+          .orElse(null);
+
       if (cachedInfo == null || cachedInfo.getDomain() == null) {
         return true;
+      } else if (currentRetryCount > DomainPresence.getDomainPresenceFailureRetryMaxCount()) {
```

Comment:
This PR is missing behavior that resets the retry count on introspectVersion or other changes.

Reply:
There is a reset now